data/os/Debian/11.yaml

```diff
@@ -0,0 +1,3 @@
+---
+letsencrypt::plugin::dns_gandi::package_provider: apt
```

Comment:
Why do you need to set the package provider? That's rarely needed.

Reply:
You're right, I will remove it